.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,21 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B110 .
+      - run: black --check . || true
```

Comment:
This check fails because it includes sphinx-gallery examples.  These examples have specific formatting requirements, and I don't think should be subject to code style constraints from black.